### DISCUSSION TITLE
PHPStorm 2024.1 freeze regulary with active Github Co-Pilot plugin

### DISCUSSION THREAD

**Discussion by mariopulkerkinnovis** [community/community#118654] [Copilot Conversations]
### Select Topic Area

Bug

### Body

**I also reported this issue already InteliJ and they asked me if I already reported this Gihub.**

Here the same information I reported InteliJ:

**Steps to Reproduce**

* Instlall Ubuntu 23.10
* Install PHP storm via snap stable version (2024.1)
* Install Github Co-Pilot plugin latest version (1.5.2.5345)

**Expected Result**
Should not freeze.

**Actual Result**

It is impossible to work with PHPstorm. Sometimes it freeze after some minutes and sometimes after hours.

Mostly it freeze when the autocomplete pop up starts to open. The whole IDE freeze. The CPU uage stays normal (no peaks).

I don't know what it is but it seems to be related to the latest Co-Pilot update and latest PHPstorm update.

When I deactivate the Co-Pilot plugin all works fine.

Also tried to deactivate all AI/ML plugins from phpstorm and activated the Co-Pilot plugin. But this results also in a freeze.

Also tried nightly builds of Co-Pilot plugin, but did not help.

I switched back to 2023.2.5 with Co-Pilot plugin 1.5.2.5345 everythink works fine.

**Comment by dimer47** [comment]
Same bug after updating to 2024.1.
I have tried to disable GitHub Copilot, but the freeze bug persists.

I switched back to 2023.3.6 so I can work.

**Reply by mariopulkerkinnovis** [reply]
With 2023.3.6 it also works for me.

**Comment by lukasion** [comment]
+1, same bug after updating to 2024.1.

**Comment by dimer47** [comment]
JetBrains has released a new preview version 2024.4.1. : https://youtrack.jetbrains.com/articles/WI-A-231736112/PhpStorm-2024.1.1-Preview-241.15989.22-build-Release-Notes?utm_source=product&utm_medium=link&utm_campaign=TBA

I just installed it, I have no problems at the moment, I will test it.

**Reply by mariopulkerkinnovis** [reply]
Does it solve the issue?
I have also started some days ago already a bug report for phpstorm:
https://youtrack.jetbrains.com/issue/WI-77033

**Reply by gaelicwinter** [reply]
In PHPStorm if I undock the Copilot window and drag it to a separate monitor, it no longer freezes. I tried doing this in the same monitor but of course it blocks the content underneath, so it really only is effective when its not laid over top of the main phpstorm window. Nonetheless, this has restored usability to the plugin, in a sense.

**Comment by goodway** [comment]
+1. the same whole year long bug!  im not used Co-Pilot. But phpStorm freeze when the autocomplete pop up starts to open. And still not fixed. Problem in 2024.0 - 2024.3, on fresh install too, with additional extensions/ or not. , on any pc , macPro. In macos Sonoma 14.1.- 14.7, macos sequoia 15.1.  It is impossible to work with phpstorm 2024

**Comment by rehanawan** [comment]
It is the UI glitch in the PHPstorm. I had the similar issue which was resolved after updating the PHPStorm 2024.3

**Reply by goodway** [reply]
Glad to hear! But I think we have different problems. I wrote the previous message after my system completely froze on a freshly installed 2024.3 version (243.21565.202). Unfortunately, 2024.x is still completely unusable for me.

2023.x - no problems
